### ISSUE TITLE
WMH synthseg原始文章

### ISSUE THREAD

**Issue by qiuyuyue2022** [qiuyuyue2022/ME#18]
在异构临床 MRI 与低场便携 MRI 中定量WMH与脑体积
Quantifying white matter hyperintensity and brain volumes in heterogeneous clinical and low-field portable MRI
https://arxiv.org/html/2312.05119v2

## 结论（CONCLUSION）

作者提出了首个能在**任意分辨率、任意对比度**（包括 pMRI）下，同时分割 **WMH + 脑区 ROIs** 的方法。未来工作将包括：更真实地模拟 WMH、在更大 pMRI 队列上评估。该方法已公开，潜力在于帮助欠发达地区分析 pMRI。

## 要点总结

1. **核心技术不是“U-Net”，而是“合成数据 + 域随机化”**：让模型见过几乎所有“可能的 MRI 长相”。
2. **WMH 强度先验**（WMH 相对白质在 T1/FLAIR 的明暗方向）是 WMH 表现提升的关键。
3. 多任务学习（回归偏场 + 回归 T1 强度）帮助模型学到成像机制，降低假阳性、提升鲁棒性。
4. 对低场/厚层/临床异构数据，作者主打指标是**体积相关性**而不是 Dice，这点很符合你做队列研究的真实需求。

---

## 摘要（ABSTRACT）

做 WMH 与脑区体积的**自动分割与定量**很有价值，但现有方法通常要求**高分辨率、信噪比高的 MRI**，这使得它们难以用于临床常见的低质量/低分辨率扫描，也难以用于**低场便携 MRI（pMRI）**。
结果就是：在资源不足地区原本很有潜力的 pMRI，反而难以支持大规模 WMH 与萎缩随访。

作者提出一种方法：
**无需重新训练（no retraining）**，即可对**任何分辨率、任何对比度**（包括 pMRI）扫描，同时分割 **WMH + 36 个脑区**。

作者在 8 个公开数据集和 1 个私有数据集上验证（该私有数据集每个受试者同时有 3T 高场与 64mT 低场扫描配对）。结果显示：

* 高低场之间估计的 **WMH 体积相关性 ρ = 0.85**
* **海马体体积相关性 ρ = 0.89**

该方法已作为 FreeSurfer 的一部分公开发布：**WMH-SynthSeg**。

---

## 1. 引言（INTRODUCTION）

近年便携 MRI（pMRI）越来越可用且成本更低，适合在社区/欠发达地区大规模应用。比如 Hyperfine 的 64mT Swoop 系统。
此外，即便不是 pMRI，临床 MRI常用 **2D 扫描**（层厚大、切片少），方向也更杂。
必须有可靠的自动分割与定量（手工在 3D 里描 ROI 不现实，也不可重复）。

目前已有很多 WMH/脑区分割方法：
* 脑区：FreeSurfer、FSL
* WMH：LST、BIANCA
* 同时分 WMH + 解剖：SAMSEG
* 深度学习（CNN）：QuickNat、FastSurfer 等用于脑区；也有 CNN 用于 WMH

但这些方法多为 常规高场 MRI（1.5–3T）设计，往往对分辨率（常见要求 1mm 各向同性）、序列（T1 做解剖、FLAIR 做 WMH）等有前提。因此它们在真实临床场景中遇到扫描方向/分辨率/对比度的大变异时性能会下降；在 pMRI 中，这个问题更严重（SNR 更低、体素更大、而且由于扫描仪几何导致远离中心处信号明显丢失）。虽然“域适配（domain adaptation）”能缓解一部分，但作者强调：

> **一个无需重训、可吃下任意对比度与分辨率 MRI 的 CNN**非常理想。

因此作者提出 **WMH-SynthSeg**：基于“域随机化（domain randomization）”思想的 CNN，可从任意分辨率/对比度扫描中分割 WMH 与脑解剖结构，包含 pMRI。

相较于作者之前能同时分割 WMH + 解剖的旧方法，该方法的改进点是：

1. **不需要重训**
2. 使用专门的 WMH 模型和“复合损失”提高敏感性与特异性
3. 明确适配低场 MRI
4. 引入多任务学习增强鲁棒性
   结果：能在临床 MRI 与 pMRI 上稳定分割 WMH 与解剖结构。

---

### 2.1 合成训练数据（Synthetic training data）

WMH-SynthSeg 依赖一个合成 MRI 生成器，类似作者以前的工作。它需要训练数据：

* N 个 **1mm 各向同性 T1**（记为 In）
* 对应的 **3D 分割标签**（记为 Sn）
* 标签包含：**脑区 ROI + WMH**

训练时，每次迭代都会生成一对“合成影像 + 合成标签”，流程如下：

1. 随机选一个样本对 (In, Sn)
2. 对 (In, Sn) 做非线性形变增强
3. 根据标签在每个体素上独立采样一个“高斯混合模型（GMM）”的强度：每个组织类别的均值/方差从均匀分布随机抽取（WMH 类别会特殊处理，后面讲）
4. 对图像加入随机平滑的偏场（bias field）
5. 随机模拟采集方向与分辨率：通过平滑/降采样生成低分辨率扫描
6. 再把低分辨率扫描上采样回 1mm 网格

最终生成：

* 上采样后的合成扫描 (I_{syn})
* 形变后的分割 (S)
* 形变后的真实图像 (I)（作为一个回归目标）
* 偏场 (B)（作为另一个回归目标）
  这些都定义在原始 1mm 网格上。

作者说：相比旧工作，这个生成器有 4 个关键改进：

**改进 (i)：WMH 强度的方向性先验（非常关键）**
WMH 的均值不再在 0–255 任意乱抽，而是模拟现实中的“相对白质的明暗关系”：

* 当白质（WM）均值较高（>128）时，说明更像 **T1-like**：WMH 均值必须**低于**白质（表现为低信号）
* 当白质均值较低（<128）时，说明更像 **T2/FLAIR-like**：WMH 均值必须**高于**白质（表现为高信号）

**改进 (ii)：噪声与偏场更强**
把噪声（强度方差）和偏场强度设为旧工作的 2 倍，以适应 pMRI 的低 SNR 与强信号丢失。

**改进 (iii)：生成器额外输出两个“回归目标”**
除了合成图 (I_{syn})，还输出形变真实图 (I) 与偏场 (B)，用于多任务学习的回归目标；实验显示这能提升鲁棒性。

**改进 (iv)：覆盖更广的分辨率/采集形态**

* 25%：1mm 各向同性（支持高分辨率）
* 25%：模拟临床 2D：平面内 1mm，层厚/层间距随机 2.5–8.5mm，方向随机
* 25%：模拟 Hyperfine Swoop 默认序列：轴位、平面内约 1.5mm、层间距 5mm
* 25%：模拟更“各向同性”的低场扫描：三方向体素 2–5mm 随机

（这段对你很重要：它解释了为什么 WMH-SynthSeg 对 2D/厚层/低场更稳。）

---

### 2.2 模型结构与训练（Model architecture and training）

模型是一个 **3D U-Net**：

* 5 个尺度层级
* 每层 64 个特征通道
* 用 **Group Normalization**
* 每层 2 次 3×3×3 卷积 + ReLU

最后一层输出通道数是 **L + 2**：

* 前 L 个通道对应各个标签类别（脑区 + WMH），进入 softmax 得到“软分割”
* 后两个通道分别预测：

  1. 偏场 ( \hat{B} )
  2. 高分辨率 T1 强度 ( \hat{I} )

训练用 Adam 优化，损失函数由 4 项等权组成：

1. 分割交叉熵 CE
2. 分割 Dice（平均 Dice）
3. ( \ell_1 ) 强度回归误差 (|I - \hat{I}|)（先把 WM 的中位强度归一到 1）
4. 偏场回归误差 (|\log B - \log \hat{B}|)

作者特别解释：为什么 Dice 要和交叉熵一起用？

* 训练初期 Dice 的梯度很“平”，不如 CE 提供稳定梯度
* 更重要：当某些扫描本来**没有 WMH**时，仅用 Dice 会出现“无论你怎么预测，WMH Dice 都可能很差/不敏感”的问题；CE 能显式惩罚假阳性（FP）

测试时：

* 把输入扫描重采样到 1mm 各向同性后输入网络
* 做测试时增强：左右翻转一次，输出翻回后与原输出平均
* 偏场与 T1 强度输出是“有用副产物”，本文中不使用

训练用 160³ 的 patch；验证损失一般在约 (10^5) 次迭代收敛（原文写法如此）。

---

### 3.1 数据集（Datasets）

共 9 个数据集：

* HCP（训练）：897 个 1mm 扫描（年轻健康），用 FreeSurfer 自动分 36 ROI
* ADNI（训练）：1148 个 1mm 扫描，用 FreeSurfer 分 ROI 与 WMH
* GE3T（训/测）：WMH challenge 子集 20 例：1mm T1 + 1×1×3mm 轴位 FLAIR；用 T1 的 FreeSurfer 解剖 + FLAIR 的手工 WMH 合并成真值
* Singapore（训/测）：challenge 另一个站点 20 例（采集同上）
* Utrecht（训/测）：challenge 第三站点 20 例
* ISBI（训/测）：15 例：1mm T1（FS 解剖）+ 1×1×2mm 轴位 FLAIR（手工 WMH）
* FLI-IAM（训/测）：15 例，T1/FLAIR 分辨率接近 1mm，7 名标注者的一致 WMH 真值
* ADHD（测试）：20 个儿童/青少年对照，1mm T1，无 WMH（用于评估假阳性）
* MGH（测试）：12 个 MS 患者：

  * 高场：1mm T1 + FLAIR
  * 低场 pMRI：轴位 T1 + FLAIR（平面内 1.6–1.8mm，层间距 5–6mm）

---

### 3.2 对比方法（Competing methods）

对比对象：

1. **SAMSEG**：贝叶斯方法、对比度自适应，作者认为它是现有方法里少数能较自然处理多种序列、同时分解剖与 WMH 的方法
2. **LST-LPA**：在 FLAIR 上表现很好，但不能处理其他对比度（所以在只 T1 的 ADHD 上无法用）
   此外做了两种消融实验（ablation）：

* 只用 Dice（类似旧方法）
* 去掉 WMH 强度均值先验（也就是去掉“WMH 相对白质明暗方向”的约束）

并提示：LST 和 SAMSEG 在原始分辨率上工作，而 WMH-SynthSeg 输出始终是 1mm 网格分割。

---

### 3.3 实验设计（Experimental setup）

三组实验：

**实验 1：Dice 分割精度**

* 用部分数据集训练、另一些数据集测试，做交叉组合
* 注意：训练输入都是合成图像；真实图像只是作为回归目标（I 与 B）

**实验 2：假阳性率（FPR）**

* 用 ADHD 青少年对照（理论上无 WMH）
* 用“预测的 WMH 体积”作为假阳性代理指标
* 训练用实验 1 的全部数据

**实验 3：低场 pMRI 能力**

* 用 MGH 数据集
* 高场真值：

  * 解剖：高场 T1 用 FreeSurfer
  * WMH：高场 FLAIR 用 LST（作为 WMH 真值）
* 因为高低场之间难做准确非线性配准（低场畸变），所以不用 Dice，而用：

  > 高场与低场估计体积之间的**相关性**来评估

---

### 3.4 结果（Results）

**表 1：高场数据的平均 Dice（解剖 + WMH）**
（解剖 Dice 是 23 个代表 ROI 的平均，WMH 单独一个 Dice。Dice指的是”比较自动分割的 WMH mask 和人工标注的 WMH mask，在空间上有多重合“）

<img width="410" height="239" alt="Image" src="https://github.com/user-attachments/assets/0c899ff5-76e0-4e1b-bb08-04edf38e7245" />

总体结论：WMH-SynthSeg（完整版）在解剖与 WMH 上都优于 SAMSEG 与 LST。

* 解剖（T1）：Dice 约 0.85
* WMH（FLAIR）：Dice 约 0.62（高于 SAMSEG 与 LST）
* 去掉 WMH 强度先验会明显降低 WMH Dice，说明“先验”对 WMH 最关键
* 还强调：它甚至能从 **T1** 得到可用的 WMH 分割（在真实世界里很实用）

**图 1：定性对比（Singapore 样例）**

<img width="1603" height="537" alt="Image" src="https://github.com/user-attachments/assets/058ebb03-e337-4f93-aaa0-05f27b7519bc" />

* LST：原分辨率输出很“脆”，但在透明隔附近（室旁）出现很多假阳性
* SAMSEG：受 partial volume 影响大，皮层等结构容易欠分割；WMH 也常欠分割
* WMH-SynthSeg：输出各向同性分割，对解剖和 WMH 都更平衡

**实验 2：假阳性（ADHD 对照）**

* WMH-SynthSeg 平均预测 WMH 体积约 950 mm³
* SAMSEG 约 877 mm³（相近）
* 去掉先验/多任务会显著升高（到 1150、1850 mm³），说明这些组件确实减少假阳性

**表 2：低场 pMRI 与高场真值的体积相关性（MGH 数据集）**

<img width="391" height="150" alt="Image" src="https://github.com/user-attachments/assets/03749af0-956d-4cf3-bdd6-a0fb5484d9c5" />

* LST 在低场完全失败（因为不是为低场设计的）
* SAMSEG 相关性中等（约 0.63–0.71）
* WMH-SynthSeg 相关性很强：

  * 海马体体积（左右平均）：ρ 约 0.89（T1 输入）/ 0.86（FLAIR 输入）
  * WMH 体积：ρ 约 0.75（T1）/ 0.85（FLAIR）

**图 2：高场与低场的示例**

<img width="1661" height="908" alt="Image" src="https://github.com/user-attachments/assets/8b62ce72-c4f2-4e62-8d7e-631c44447c6b" />

* (a)(b) 高场 FLAIR 与 LST WMH（当作真值）
* (c)(d) 高场 T1 与 FreeSurfer 解剖（当作真值）
* (e)(f) 低场 pMRI 与 WMH-SynthSeg 输出
  并提醒：即使做了仿射配准，由于低场的非线性畸变，切片上解剖仍会有差异，这也是他们用“体积相关”而非 Dice 的原因。
